library/ortools/rust/BUILD

```diff
@@ -26,6 +26,7 @@ rust_library(
     srcs = ["ortools.rs"],
     deps = [
         "@vaticle_dependencies//library/crates:cxx",
+        "@vaticle_dependencies//library/crates:libc",
```

Comment:
`ortools` depends transitively on `libc`, but this dependency isn't being picked up correctly in an `aarch64` architecture environment, so we now depend on it explicitly.